native/src/main/java/io/ballerina/stdlib/grpc/builder/BallerinaFileBuilder.java

```diff
@@ -66,6 +66,7 @@
     public static Map<String, String> enumDefaultValueMap = new HashMap<>();
     public static Map<String, Boolean> dependentValueTypeMap = new HashMap<>();
     public static Map<String, Class> streamClassMap;
+    public static Map<String, List<String>> unusedImports;
```

Comment:
Shall we make it a private non-static variable?

Reply:
Updated in [2fff30a](https://github.com/ballerina-platform/module-ballerina-grpc/pull/739/commits/2fff30aae3b80dd58b44b9fa526700c07c28f562)